### PR TITLE
Add rebalance doc

### PR DESCRIPTION
**Pull request by bincircon**

### CODE REVIEW

File: api/rebalance-deactivation.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Abort The Current In Progressing Topology Rebalance
```
Comment (rileyberton) [inline api/rebalance-deactivation.md]: "In Progress"

File: rebalance-apis.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Rebalance APIs and procedure 
+
+Procedure
+
+For example, we have two snow clusters. First cluster has two nodes with write factor of 2. Second cluster has single node with write factor of 1.
```
Comment (rileyberton) [inline rebalance-apis.md]: let's use "IRONdb" instead of "snow"

File: rebalance-apis.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+We want to combine these three nodes together to create a three node cluster with write factor of 3.
+
+- Combine first cluster topology and second cluster topology xml and create new topology xml with three nodes
+- Snow import the new topology and create new topology uuid
```
Comment (rileyberton) [inline rebalance-apis.md]: `snowthimport`.. also we should given example command line here.

File: rebalance-apis.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+- Load the new topology to all three nodes via POST /topology/[new topology uuid] with new topology xml
```
Comment (rileyberton) [inline rebalance-apis.md]: a `curl` command would be helpful here.

**Comment by bincircon** [comment]
@rileyberton addressed the review comments

**Comment by rileyberton** [comment]
Looks good to me

**Pull request merged**